src/database/nullDatabase.js

```diff
@@ -0,0 +1,43 @@
+
+class NullDatabase {
+
+  constructor(_dbFilename, events, cb) {
```

Comment:
Constructor can be removed, since it does nothing.

Reply:
Since it seems to take a callback I opted to leave it as it is to avoid any hanging issues =